java/lance-namespace-adapter/src/main/java/com/lancedb/lance/namespace/adapter/ClientToServerResponse.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.lancedb.lance.namespace.adapter;
+
+import com.lancedb.lance.namespace.server.springboot.model.GetNamespaceResponse;
+import com.lancedb.lance.namespace.server.springboot.model.GetTableResponse;
+import com.lancedb.lance.namespace.server.springboot.model.ListNamespacesResponse;
+import com.lancedb.lance.namespace.server.springboot.model.RegisterTableResponse;
```

Comment:
Can we find a way to put the req/resp entities under the same packages?

Reply:
need to do some experiments with that, let me check if that is feasible

Reply:
https://openapi-generator.tech/docs/generators/java

https://openapi-generator.tech/docs/generators/spring

Looks like there is not a way to let them be the same model, because both would generate models... so some sort of conversion is necessary to work between server and client models although they are exactly the same.

Reply:
In that case we probably need to add some enforcement that spec changes must also update client and server model mapping.

Reply:
Added https://github.com/lancedb/lance-namespace/issues/72 to track this